2025-03-12 19:26:31,042 - image_classifier_api - INFO - Loading model...
2025-03-12 19:26:31,044 - image_classifier_api - ERROR - Failed to load model: SavedModel file does not exist at: models/intel_classifier_deployment\{saved_model.pbtxt|saved_model.pb}
2025-03-12 23:45:18,307 - image_classifier_api - INFO - Loading model...
2025-03-12 23:45:23,840 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-12 23:45:28,125 - image_classifier_api - INFO - Loading model...
2025-03-12 23:45:33,522 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-12 23:45:33,569 - image_classifier_api - INFO - Loading model...
2025-03-12 23:45:38,833 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 00:09:42,529 - image_classifier_api - INFO - Loading model...
2025-03-13 00:09:48,301 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 00:09:48,346 - image_classifier_api - INFO - Loading model...
2025-03-13 00:09:53,923 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 00:14:24,843 - image_classifier_api - INFO - Loading model...
2025-03-13 00:14:30,365 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 00:14:34,483 - image_classifier_api - INFO - Loading model...
2025-03-13 00:14:40,140 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 00:14:40,181 - image_classifier_api - INFO - Loading model...
2025-03-13 00:14:45,490 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 00:15:36,773 - image_classifier_api - INFO - Prediction request received from user: admin
2025-03-13 00:16:24,143 - image_classifier_api - INFO - Loading model...
2025-03-13 00:16:29,425 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 00:16:29,459 - image_classifier_api - INFO - Loading model...
2025-03-13 00:16:34,795 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 00:16:34,808 - image_classifier_api - INFO - Prediction request received from user: admin
2025-03-13 00:18:46,346 - image_classifier_api - INFO - Prediction request received from user: admin
2025-03-13 00:18:47,242 - image_classifier_api - INFO - Predicted class: street with confidence: 1.0000
2025-03-13 00:18:50,324 - image_classifier_api - INFO - Prediction request received from user: admin
2025-03-13 00:18:50,397 - image_classifier_api - INFO - Predicted class: street with confidence: 1.0000
2025-03-13 00:19:17,690 - image_classifier_api - INFO - Prediction request received from user: admin
2025-03-13 00:19:17,765 - image_classifier_api - INFO - Predicted class: mountain with confidence: 0.8668
2025-03-13 00:52:19,186 - image_classifier_api - INFO - Loading model...
2025-03-13 00:52:19,218 - image_classifier_api - ERROR - Failed to load model: File format not supported: filepath=models/intel_classifier_deployment. Keras 3 only supports V3 `.keras` files and legacy H5 format files (`.h5` extension). Note that the legacy SavedModel format is not supported by `load_model()` in Keras 3. In order to reload a TensorFlow SavedModel as an inference-only layer in Keras 3, use `keras.layers.TFSMLayer(models/intel_classifier_deployment, call_endpoint='serving_default')` (note that your `call_endpoint` might have a different name).
2025-03-13 01:03:25,507 - image_classifier_api - INFO - Loading model...
2025-03-13 01:03:31,010 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 01:03:35,257 - image_classifier_api - INFO - Loading model...
2025-03-13 01:03:40,725 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 01:03:40,768 - image_classifier_api - INFO - Loading model...
2025-03-13 01:03:46,176 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 01:07:24,732 - image_classifier_api - INFO - Prediction request received from user: admin
2025-03-13 01:07:25,630 - image_classifier_api - INFO - Predicted class: glacier with confidence: 0.9985
2025-03-13 01:09:51,197 - image_classifier_api - INFO - Prediction request received from user: admin
2025-03-13 01:09:51,650 - image_classifier_api - INFO - Predicted class: sea with confidence: 0.6096
2025-03-13 01:11:28,009 - image_classifier_api - INFO - Prediction request received from user: admin
2025-03-13 01:11:28,100 - image_classifier_api - INFO - Predicted class: glacier with confidence: 0.9984
2025-03-13 02:16:25,501 - image_classifier_api - INFO - Loading model...
2025-03-13 02:16:25,522 - image_classifier_api - ERROR - Failed to load model: File format not supported: filepath=models/intel_classifier_deployment. Keras 3 only supports V3 `.keras` files and legacy H5 format files (`.h5` extension). Note that the legacy SavedModel format is not supported by `load_model()` in Keras 3. In order to reload a TensorFlow SavedModel as an inference-only layer in Keras 3, use `keras.layers.TFSMLayer(models/intel_classifier_deployment, call_endpoint='serving_default')` (note that your `call_endpoint` might have a different name).
2025-03-13 02:23:22,843 - image_classifier_api - INFO - Loading model...
2025-03-13 02:23:22,857 - image_classifier_api - ERROR - Failed to load model: File format not supported: filepath=models/intel_classifier_deployment. Keras 3 only supports V3 `.keras` files and legacy H5 format files (`.h5` extension). Note that the legacy SavedModel format is not supported by `load_model()` in Keras 3. In order to reload a TensorFlow SavedModel as an inference-only layer in Keras 3, use `keras.layers.TFSMLayer(models/intel_classifier_deployment, call_endpoint='serving_default')` (note that your `call_endpoint` might have a different name).
2025-03-13 02:33:03,467 - image_classifier_api - INFO - Loading model...
2025-03-13 02:33:03,753 - tensorflow - WARNING - From C:\Users\Sourik\anaconda3\envs\tf_env\lib\site-packages\keras\src\saving\legacy\saved_model\load.py:107: The name tf.gfile.Exists is deprecated. Please use tf.io.gfile.exists instead.

2025-03-13 02:33:03,797 - tensorflow - WARNING - From C:\Users\Sourik\anaconda3\envs\tf_env\lib\site-packages\keras\src\engine\functional.py:156: The name tf.executing_eagerly_outside_functions is deprecated. Please use tf.compat.v1.executing_eagerly_outside_functions instead.

2025-03-13 02:33:08,870 - absl - INFO - Fingerprint not found. Saved model loading will continue.
2025-03-13 02:33:08,871 - absl - INFO - path_and_singleprint metric could not be logged. Saved model loading will continue.
2025-03-13 02:33:08,900 - tensorflow - WARNING - From C:\Users\Sourik\anaconda3\envs\tf_env\lib\site-packages\keras\src\layers\normalization\batch_normalization.py:979: The name tf.nn.fused_batch_norm is deprecated. Please use tf.compat.v1.nn.fused_batch_norm instead.

2025-03-13 02:33:09,455 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 02:33:15,506 - image_classifier_api - INFO - Loading model...
2025-03-13 02:33:15,801 - tensorflow - WARNING - From C:\Users\Sourik\anaconda3\envs\tf_env\lib\site-packages\keras\src\saving\legacy\saved_model\load.py:107: The name tf.gfile.Exists is deprecated. Please use tf.io.gfile.exists instead.

2025-03-13 02:33:15,857 - tensorflow - WARNING - From C:\Users\Sourik\anaconda3\envs\tf_env\lib\site-packages\keras\src\engine\functional.py:156: The name tf.executing_eagerly_outside_functions is deprecated. Please use tf.compat.v1.executing_eagerly_outside_functions instead.

2025-03-13 02:33:21,082 - absl - INFO - Fingerprint not found. Saved model loading will continue.
2025-03-13 02:33:21,083 - absl - INFO - path_and_singleprint metric could not be logged. Saved model loading will continue.
2025-03-13 02:33:21,135 - tensorflow - WARNING - From C:\Users\Sourik\anaconda3\envs\tf_env\lib\site-packages\keras\src\layers\normalization\batch_normalization.py:979: The name tf.nn.fused_batch_norm is deprecated. Please use tf.compat.v1.nn.fused_batch_norm instead.

2025-03-13 02:33:21,723 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 02:33:21,766 - image_classifier_api - INFO - Loading model...
2025-03-13 02:33:27,047 - absl - INFO - Fingerprint not found. Saved model loading will continue.
2025-03-13 02:33:27,048 - absl - INFO - path_and_singleprint metric could not be logged. Saved model loading will continue.
2025-03-13 02:33:27,645 - image_classifier_api - INFO - Model loaded successfully with classes: ['buildings' 'forest' 'glacier' 'mountain' 'sea' 'street']
2025-03-13 02:35:03,067 - image_classifier_api - INFO - Prediction request received from user: admin
2025-03-13 02:35:04,265 - image_classifier_api - INFO - Predicted class: sea with confidence: 0.6182
